src/components/atoms/inputs/TextArea.tsx

```diff
@@ -0,0 +1,49 @@
+import { type FieldError } from "react-hook-form"
+
+type Props = {
+  className?: string;
+  isRequired?: boolean;
+  label: string;
+  defaultValue?: string;
+  placeholder?: string;
+  value?: string;
+  error?: FieldError | undefined;
+  id?: string;
+  height?: string;
+};
+
+function TextArea(props: Props) {
+  const {
+    id,
+    className,
+    isRequired = false,
+    label,
+    defaultValue,
+    placeholder,
+    value,
+    error,
+    height,
+  } = props;
+
+
+  return (
+    <div className={`flex flex-col ${className}`}>
+      <div className="flex flex-row gap-[5px] mb-2">
+        <label className="text-[16px] text-text_bold font-bold">{label}</label>
+        {isRequired && <span className="text-[16px] font-bold text-tertiary">*</span>}
+      </div>
+      <input
```

Comment:
Its Better to use textArea tag insetead of input while making a textArea bucasue in input feild we can't go next line 
reference : 
<textarea id="w3review" name="w3review" rows="4" cols="50">
At w3schools.com you will learn how to make a website. They offer free tutorials in all web development technologies.
</textarea>